tools/testoperator/src/tests/http/consistency.rs

```diff
@@ -66,13 +68,13 @@ pub async fn consistency_run(args: &HttpConsistencyTestArgs) -> anyhow::Result<(
     let results = test.metrics()?;
 
     let mut spiced_instance = test.end();
-    spiced_instance.stop()?;
+
+    print_ascii_table(&results);
```

Comment:
```suggestion
    let metrics = test.collect()?;
    metrics.show()?;
    let (p50, p95): (Vec<f64>, Vec<f64>) = metrics.metrics.iter()...
```

I don't think we need to make a custom ascii table printer. We can use arrow to show the dataframe. Would need a little re-work to the p50, p95 iterator to use the inner metrics from the return of `test.collect()`